src/de/gurkenlabs/litiengine/gui/screens/ScreenManager.java

```diff
@@ -1,8 +1,6 @@
 package de.gurkenlabs.litiengine.gui.screens;
 
-import java.util.Collection;
-import java.util.List;
-import java.util.Optional;
+import java.util.*;

```

Comment:
I don't recommend using wildcard imports for any packages you don't control. See https://stackoverflow.com/questions/147454/why-using-a-wild-card-with-a-java-import-statement-bad/150315#150315.

Reply:
Yeah, sorry. I tend to trust my IDE with the import statements. I only touch those when something is wrong, or there are lingering unused imports.